ModelBundle/DataFixtures/MongoDB/LoadNodeDemoData.php

```diff
@@ -26,6 +26,36 @@ class LoadNodeDemoData extends AbstractFixture implements OrderedFixtureInterfac
      */
     public function load(ObjectManager $manager)
     {
+
+        $manager->getConnection()->initialize();
+        $dataBase = $manager->getDocumentDatabase('OpenOrchestra\ModelBundle\Document\Node');
+        $dataBase->execute("
+db.node.createIndex( { nodeId: 1, siteId: 1, language: 1, version: 1 }, { unique: true } )
+db.system.js.save(
+    {
+        _id : 'duplicateNode' ,
+        value : function (data){
+            var node = null
+            while (1) {
```

Comment:
In case of JMeter tests 3 can't be enough. Which value could we use in this case? @itkg-ngilain